### DISCUSSION TITLE
Custom permalink for the RSS entries

### DISCUSSION THREAD

**Discussion by robotfx** [FreshRSS/FreshRSS#3714] [Q&A] [answered]
Hi,

Is it possible to add the source (ie get=f_196) in the rss.phtml file, so it will appear in the site's feed?
What I am trying to accomplish is to have the site's RSS entries links point to the article, something like this
`?a=normal&get=f_196#1626952321430346
`
instead of pointing to the original article link.

In the rss.phtml file I can only get the article id from the <?= $item->id() ?>, but cannot see any way to get the feed source number (the get variable above, f_196). So this is what I have so far in rss.phtml:
`<link>https://site.com/stiri/?a=norma&get=f_XXX&#<?= $item->id() ?></link>
`
How to get the proper source number instead of XXX?

Thank you.

**Comment by Alkarex** [comment, marked as answer]
Try to see `FreshRSS_Context::currentGet(true)`
https://github.com/FreshRSS/FreshRSS/blob/edge/app/Models/Context.php#L149

An RSS feed can be created from any FreshRSS view, and not only a feed. For instance a category, or a global search.

**Reply by robotfx** [reply]
Thank you, I've managed to do it like this:
`<link>kttps://site.com/stiri/?a=normal&get=f_<?= $item->feed() ?>#<?= $item->id() ?></link>
`
Now I have another issue: opening the link in browser does not jump to the hash. I keep looking for javascript solutions but haven't found anything yet. Any idea how to accomplish this?
